src/v2/SuperstateToken.sol

```diff
@@ -0,0 +1,463 @@
+// SPDX-License-Identifier: BUSL-1.1
+pragma solidity ^0.8.26;
+
+import {ERC20Upgradeable} from "openzeppelin-contracts-upgradeable/token/ERC20/ERC20Upgradeable.sol";
+import {Ownable2StepUpgradeable} from "openzeppelin-contracts-upgradeable/access/Ownable2StepUpgradeable.sol";
+import {PausableUpgradeable} from "openzeppelin-contracts-upgradeable/security/PausableUpgradeable.sol";
+import {ECDSA} from "openzeppelin-contracts/utils/cryptography/ECDSA.sol";
+
+import {console} from "forge-std/console.sol";
```

Comment:
remove

Reply:
Addressed in
- https://github.com/superstateinc/ustb/pull/56/commits/4e52d672532d51249bce9677ae1801a91404eda8
- https://github.com/superstateinc/ustb/pull/56/commits/863dda0005df872dec9136d026df07f918dbb2e5